pkg/proc/gdbserial/gdbserver.go

```diff
@@ -363,8 +361,8 @@ func LLDBLaunch(cmd []string, wd string, foreground bool, debugInfoDirs []string
 	var listener net.Listener
 	var port string
 	var process *exec.Cmd
-	if _, err := os.Stat(GetDebugServerAbsolutePath()); err == nil {
-		listener, err = net.Listen("tcp", "127.0.0.1:0")
+	if debugserverExecutable := getDebugServerAbsolutePath(); debugserverExecutable != "" {
+		listener, err := net.Listen("tcp", "127.0.0.1:0")
```

Comment:
This is the cause of the breakage. This variable shadows the `listener` variable above, causing it to never be set, thus we end up with a zero-value listener even though we were able to find the debug server executable.

Reply:
Thx, done